react-native/README.md

```diff
@@ -0,0 +1,54 @@
+
+[PowerPoint Template and Craft Demo Outline](https://github.com/QuickBase/interview-demos/blob/master/QuickBase_CraftDemo_PresentationTemplate.pptx)
+
+One of our Quick Base partners has asked us to create a feature inside our react-native application. 
+After an internal review, we have found a significant portion of our user base that could benefit from their feature. 
+You have been given the opportunity to create a proof of concept. 
+
+## General Description
+
+The purpose of the craft demo is to give us a jumping-off-point for discussion of your approach to solving a problem and of your code. We are interested in learning how you approach this problem and go about solving it.
+
+We expect this project to take experienced developers about two to four hours to complete. We know not everyone has unlimited time to work on interview projects and we do not expect that everyone completes all of the requirements. However, you may be asked how you might have solved a requirement you missed had you been able to work on this project without time constraints. Additionally, if you want to extend the project beyond the listed requirements (e.g. completing some of the stretch goals, or being creative with your own requirements), please feel free to do so as you see fit. Please note that we prefer higher-code quality over additional features if you start to run out of time.
+
+Use of your favorite libraries, SDKs and frameworks is welcomed but optional. The project outlined below can be completed with or without these tools.
+
+Please be prepared to demo your solution in a working development environment on the day of your interview. When you present your craft demo, we will ask you to describe how you arrived at your solution. After that, we will ask for a tour of your code with a focus on how you arrived at the core functionality and any design decisions and/or trade-offs you made during the process.
+
+## User story
+Sally's garage manages a high volume of inventory for many independent auto mechanics using a Quick Base application. 
+Sally is hoping to use a native iOS application to add new models to their inventory tracking Quick Base app. 
+
+## Technical Requirements
+
+1. Clone our demo app from the App Exchange (see the [FAQ](#faq))
+1. Recreate a React native version of the `create new part` form and post the data via the quick base api to your app.
+    1. To get to the create new record view, go to `My Apps`
+    1. Under my apps, select `React Native Craft Demo`
+    1. After selecting your app, navigate to `Parts`
+    1. Click `+ New Part` 
+1. Create a `react-native` view that pushes data to the Quick Base API
```

Comment:
I would remove the line above this comment and only use the text below for this bullet point. By repeating the words "Create a react-native view", it sounds like there should be two views. But here is only one.